Firestore/Source/Local/FSTLRUGarbageCollector.h

```diff
@@ -26,6 +29,22 @@
 
 extern const firebase::firestore::model::ListenSequenceNumber kFSTListenSequenceNumberInvalid;
 
+struct FSTLruGcParams {
```

Comment:
Note that you can do so inline here without adding any extra files. It would be:

```c++
namespace firebase {
namespace firestore {
namespace local {

struct LruParams {  // or LruOptions, whatever
  // members...
};

}  // namespace local
}  // namespace firestore
}  // namespace firebase
```